### ISSUE TITLE
Fix clipboard module related crashes

### ISSUE THREAD

**Issue by matejkriz** [trezor/trezor-suite#12594]
https://play.google.com/console/u/0/developers/4710771878548015996/app/4973792525499935694/vitals/crashes/0875d2cc13f59ef7aeb02c6643969aa4/details?days=28&versionCode=55&isUserPerceived=true

```
Exception kotlin.UninitializedPropertyAccessException: lateinit property clipboardEventEmitter has not been initialized
  at expo.modules.clipboard.ClipboardModule$definition$lambda$12$$inlined$OnActivityEntersForeground$1.invoke (ModuleDefinitionBuilder.kt:144)
  at expo.modules.clipboard.ClipboardModule$definition$lambda$12$$inlined$OnActivityEntersForeground$1.invoke (ModuleDefinitionBuilder.kt:96)
  at expo.modules.kotlin.events.BasicEventListener.call (EventListener.kt:13)
  at expo.modules.kotlin.ModuleHolder.post (ModuleHolder.kt:94)
  at expo.modules.kotlin.ModuleRegistry.post (ModuleRegistry.kt:84)
  at expo.modules.kotlin.AppContext.onHostResume$expo_modules_core_release (AppContext.kt:318)
  at expo.modules.kotlin.ReactLifecycleDelegate.onHostResume (ReactLifecycleDelegate.kt:21)
  at com.facebook.react.bridge.ReactContext.onHostResume (ReactContext.java:308)
  at com.facebook.react.ReactInstanceManager.moveToResumedLifecycleState (ReactInstanceManager.java:765)
  at com.facebook.react.ReactInstanceManager.moveReactContextToCurrentLifecycleState (ReactInstanceManager.java:800)
  at com.facebook.react.ReactInstanceManager.lambda$setupReactContext$3 (ReactInstanceManager.java:1174)
  at com.facebook.react.ReactInstanceManager.$r8$lambda$iHpnDjhPTftdZc2HP9iD9hTDvL4
  at com.facebook.react.ReactInstanceManager$$ExternalSyntheticLambda0.run
  at android.os.Handler.handleCallback (Handler.java:958)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loopOnce (Looper.java:230)
  at android.os.Looper.loop (Looper.java:319)
  at android.app.ActivityThread.main (ActivityThread.java:8919)
  at java.lang.reflect.Method.invoke
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:578)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1103)
  ```

**Comment by vytick** [comment]
It seems, this should be fixed by upgrading expo:

as mentioned in https://github.com/expo/expo/issues/26788 
it should be fixed by https://github.com/expo/expo/pull/26944 
which was released in **expo@50.0.12**

si #12600 should probably resolve this issue

**Comment by matejkriz** [comment]
No longer reported since 24.7.3
